src/layout/AppLayout.tsx

```diff
@@ -35,7 +36,8 @@ const AppLayout = ({ children }: IAppLayout) => {
   );
   return (
     <Page mainContainerId={pageId} header={Header} skipToContent={PageSkipToContent}>
-      {children}
+      {!backendAvailable && <WaitingPage setBackendAvailable={setBackendAvailable} />}
+      {backendAvailable && children}
```

Comment:
Can we not just do `{backendAvailable ? children : <WaitingPage/>}`

Reply:
yeah we really should!